src/core/creative.js

```diff
@@ -0,0 +1,143 @@
+import { SortedListItem } from '../util/sorted-list-item'
+import { UniversalAdId } from './universal-ad-id'
+
+/**
+ * A creative.
+ *
+ * @class Creative
+ * @extends SortedListItem
+ * @author Tim De Pauw <tim.depauw@zentrick.com>
+ * @copyright © 2018 Zentrick nv
+ */
+export class Creative extends SortedListItem {
+  constructor () {
+    super()
+    this._universalAdId = new UniversalAdId()
+    this._creativeExtensions = []
+  }
+
+  // Attribute(s).
+
+  /**
+   * The ID of this creative.
+   *
+   * Bounded: 1
+   *
+   * @type {string}
+   */
+  get id () {
+    return this._id
+  }
+
+  set id (value) {
+    this._id = value
+  }
+
+  /**
+   * The ID of the {@link Ad} with which this creative is served.
+   *
+   * Bounded: 0-1 if VAST 4.0, 1 otherwise
```

Comment:
In this case, "bounded" is less clear than e.g., "required for VAST 2.0 and 3.0".